help/templates/template.cshtml

```diff
@@ -121,6 +121,7 @@
 															<ul>
 																<li><a href="/dotnet-cli.html">Cli</a></li>
 																<li><a href="/dotnet-assemblyinfo.html">AssemblyInfoFile</a></li>
+                                                                <li><a href="/apidocs/v5/fake-dotnet-fsc.html">MSBuild</a></li>
```

Comment:
You might want to change the text here to `Fsc` :)

Reply:
Doh! My bad haha, I had a feeling I was forgetting something. Thanks